app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    command_mv, file_to_opening, directory_to_writing = command.split(" ")
+    if command_mv == "mv":
+        if directory_to_writing.count("/"):
+
+            directory_to_writing_in_list = directory_to_writing.split("/")
+            file_to_writing = directory_to_writing_in_list[-1]
+            folders = ""
+            for ind in range(len(directory_to_writing_in_list) - 1):
```

Comment:
ind is a bad name for variable, use full names for naming
also u can use makedirs method instead of loop)

Reply:
Fixed